notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,2260 @@
+{
```

Comment:
<p>Maybe we can avoid putting is_tangent and to_tangent yet, since we have not talked about it yet?</p><p> </p><p>Alternatively, you could introduce the concept of tangent space to a manifold before (in the section about manifold), which would allow to talk about these methods. Your call!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Just added tangent space to manifold notebook :D</p>